dev-infrastructure/modules/common.bicep

```
@@ -102,8 +102,8 @@ var _locationAvailabilityZones = {
   eastus2euap: {
     availabilityZones: [
       '1'
```

```suggestion
      '1'
      // '2' Not available in EV2
      '3'
      '4'
```
nit: I would leave 2 commented out with a short explanation, for posterity